src/overview/comparing-std-and-no_std.md

```diff
@@ -1,69 +1,47 @@
-# Comparing `std` and `no_std`
+# Standard library (`std`) vs. Bare Metal (`no_std`)
 
-There are several factors that must be considered when choosing between `std` ([esp-idf-hal]) and `no_std` (eg. [esp-hal]). As stated previously, each approach has its own unique set of advantages and disadvantages. While we can't decide for you, this section will hopefully allow you to make an educated decision.
+There are several factors that must be considered when choosing between `std` and `no_std` approaches. Each approach has its own unique set of advantages and disadvantages. While we can't decide for you, this section will hopefully allow you to make an educated decision.
 
 [esp-idf-hal]: https://github.com/esp-rs/esp-idf-hal
 [esp-hal]: https://github.com/esp-rs/esp-hal
 
-## Application Runtimes
+See also the comparison in [The Embedded Rust Book][embedded-rust-book-intro-std]
 
-In the case of applications (as opposed to libraries) the standard library provides a runtime that handles setting up stack overflow protection, spawning the main thread before an application's `main` function is invoked, and handling of command-line arguments.
+[embedded-rust-book-intro-std]: https://docs.rust-embedded.org/book/intro/no-std.html#a-no_std-rust-environment
```

Comment:
@SergioGasquez 

Should this link be added here?

Reply:
I think its a good resource to link here